backend/models/schemas.py

```
@@ -0,0 +1,40 @@
+from pydantic import BaseModel, Field
+from typing import List, Optional
+
+class ChatRequest(BaseModel):
+    conversationId: Optional[str] = None
+    message: str
+    context: Optional[dict] = None
+
+class MatchUser(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)